src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +83,109 @@ public static Command duplicate(Command command) {
                 command.getRequirements().toArray(Subsystem[]::new)
         );
     }
+
+    /**
+     * @param isDelivery if the robot is shooting the note for a delivery or for a shot
+     * @return a command that sets the robot to shoot at the given target
+     */
+    public static Command getShootAtShootingTargetCommand(boolean isDelivery) {
```

Comment:
Add general text as well (like this:)
```suggestion

    /**
    * Creates a command that blah blah blah.
    * 
     * @param isDelivery if the robot is shooting the note for a delivery or for a speaker shot
     * @return the command
     */
    public static Command getShootAtShootingTargetCommand(boolean isDelivery) {
```

Reply:
Unresolved, look at the whole suggestion.